shared-unpackaged/get-bounds.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import type {ReferenceObject} from "popper.js";
+import type {Bounds} from "./types.js";
+import getEdges from "./get-edges.js";
+
+export default function getBounds(
+    element: Element | ReferenceObject,
+    withoutEdges: boolean = false,
```

Comment:
I am not sure why the bounds were calculated with the margin, padding, and border removed. I imagine it was to base things off the "visible" boundary of the item.

I did play around with not doing this and did not immediately notice a difference, but that doesn't mean there isn't one, so I left this as-is so that we can pick and choose.